### ISSUE TITLE
Type instability in extended_div

### ISSUE THREAD

**Issue by dpsanders** [JuliaIntervals/IntervalArithmetic.jl#312]
```
julia> @code_warntype extended_div(1..2, 3..4)
Body::Union{Nothing, Tuple{Interval{Float64},Interval{Float64}}}
```
This should be type stable!

cc @yashrajgupta

**Comment by dpsanders** [comment]
Also the code is *huge*

**Comment by dpsanders** [comment]
I think it's because of the use of the explicit `∅`. This should be `emptyinterval(T)` or something.

**Comment by dpsanders** [comment]
This is because there was no default return value at the end of the function! :/

**Comment by dpsanders** [comment]
Fixed by #313
